docs/source/api_ref_models.rst

```diff
@@ -99,7 +99,7 @@ To download the Llama3.1-405B-Instruct model:
     tune download meta-llama/Meta-Llama-3.1-405B-Instruct --ignore-patterns "original/consolidated*" --hf-token <HF_TOKEN>
 
 To download the Llama3 weights of the above models, you can instead download from `Meta-Llama-3-8B-Instruct` and
-`Meta-Llama-3-70B-Instruct`.
+`Meta-Llama-3-70B-Instruct`, and remove the ignore patterns flag.
```

Comment:
unfortunately, 8b uses meta ckpt, and 70b uses hg ckpt. Ideally, we should update their configs to all use hf, so we can always ignore consolidated. To avoid adding too much info here, i suggested that the user can just remove the ignore patterns flag, downloading all models.